ide-support/revsblibrary.livecodescript

```diff
@@ -2573,7 +2573,7 @@ private command __UpdateSettingsFromCodeFolder pPath, pCodeFolderName, pPlatform
                appendToStringList tFolderPath & slash & tLine, xSettings["extension_code_resources"]
             end repeat
             
-            if pPlatform is "mac" then
+            if pPlatform is among the items of "mac-ios" then
```

Comment:
@trevordevore perhaps use `words of "mac ios"` here

Reply:
@montegoulding Changed.